src/Components/Tabs/component.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Tabs as Base } from 'react-bootstrap';
+import Tab from '../Tab/component';
+
+type TabsProps = {
```

Comment:
if those are not used in the exactly this file, then there is no need to describe those properties

Reply:
This declaration serves as a model for the component props.

Reply:
This declaration serves as a model for the component props.